config/hyperdrive_config.toml

```diff
@@ -0,0 +1,38 @@
+title = "example simulation config"
```

Comment:
Why did you add this file? Can you just modify `simulation_config` instead? What is the difference between the two? We should be weary about this config folder blowing up for random use-cases.